### ISSUE TITLE
Elevated urine 4-hydroxyphenlyllactic acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6429]
**Preferred term label:** Elevated urine 4-hydroxyphenlyllactic acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of 4-hydroxyphenlyllactic acid in the urine is above the normal range.

**Parent term (use hpo.jax.org/app)** Elevated urinary carboxylic acid HP:0040156

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
